admin.js

```diff
@@ -100,7 +100,7 @@
     matchHook = function (moduleName, origionalMatch, config) {
         return function(event, commandPrefix) {
             if (getHasPermission(config, event.sender_id, event.sender_name, event.thread_id, moduleName)) {
-                return origionalMatch.call(this, event, commandPrefix);
+                return origionalMatch.call(exports.platform, event, commandPrefix);
```

Comment:
`exports.platform` is definately not the correct `this` context here. I suggest reviewing `platform.js` if you are fiddling with `this`.